custom/ewsghana/templates/ewsghana/partials/ews_multibar_chart.html

```diff
@@ -55,8 +55,19 @@
         multibar_chart.multibar.groupSpacing(chart_config.groupSpacing);
         multibar_chart.multibar.forceY(chart_config.forceY);
 
+        var data =  JSON.parse(JSON.stringify(multibar_chart_data));
+        var productCodes = [];
+        $("#report_table_product_selection_pane input:not(:checked)").each(function () {
```

Comment:
One other way to write this is to use `map` which essentially does what you're doing with your `each`:

```
productCodes =  $("#report_table_product_selection_pane input:not(:checked)").map(function() {
  return $(this).val();
}
```
Slightly more idiomatic